src/shared/components/payments_table/deletion_entry.js

```diff
@@ -1,25 +1,35 @@
-var request = require('xhr')
 var React = require('react')
 
 module.exports = React.createClass({
   getInitialState: function () {
-    return { confirmation: false } },
+    return { confirmation: false }
+  },
```

Comment:
Please don't do this mixed in with a PR that has some stuff going on. It makes the diff very noisy